import-scripts/generate_cdd_attribute_tsv.py

```diff
@@ -122,14 +129,23 @@ def load_new_cdd_attributes(new_attributes_file, uri_dictionary):
                 invalid_attributes.append(data[normalized_column_header_position])
             for position in range(len(header)):
                 try:
-                    new_attribute[header[position]] = insert_value(header[position], data[position], data[normalized_column_header_position])
+                    if (data[position].isascii()):
+                        new_attribute[header[position]] = insert_value(header[position], data[position], data[normalized_column_header_position])
+                    else:
+                        non_ascii_values.append([data[position], unidecode(data[position])])
```

Comment:
This will help us print out all detected non-ascii characters, but it will not keep track of which Clinical Attribute was involved. This is ok -- but we might also consider tracking the list of affected Clinical Attributes and reporting those .. so that the curators will be able to more quickly find the problem lines.